buildconfig/Jenkins/Conda/run-tests

```diff
@@ -20,6 +20,8 @@ ENABLE_DOC_TESTS=$5
 BUILD_THREADS=$6
 USE_CORE_DUMPS=$7
 
+# Run only 2 tests at a time because of memory constraints in the system tests
+BUILD_THREADS_SYSTEM_TESTS=2
```

Comment:
Can we bump this to 4?
I'm pretty sure all of our builders have 32GB+ and we use 4 on other builders